pensjonsmaler/src/main/kotlin/no/nav/pensjon/brev/maler/redigerbar/EndringAvAlderspensjonSivilstand.kt

```diff
@@ -0,0 +1,1127 @@
+
+        // Hvis etterbetaling (Selectable) - etterbetalingAP_002
+        showIf(not(vedtakEtterbetaling)) {
+            showIf(saksbehandlerValg.etterbetaling) {
+                // etterbetalingAP_002
+                title1 {
+                    text(
+                        Bokmal to "Etterbetaling",
+                        Nynorsk to "Etterbetaling",
+                        English to "Retroactive payment",
+                    )
+                }
+                paragraph {
+                    textExpr(
+                        Bokmal to "Du får etterbetalt pensjon fra ".expr() + kravVirkDatoFom + ". Etterbetalingen vil vanligvis bli utbetalt i løpet av syv virkedager. Vi kan trekke fra skatt og ytelser du har fått fra for eksempel Nav eller tjenestepensjonsordninger. Derfor kan etterbetalingen din bli forsinket. Tjenestepensjonsordninger har ni ukers frist på å kreve trekk i etterbetalingen. Du kan sjekke eventuelle fradrag i utbetalingsmeldingen på $DITT_NAV.",
+                        Nynorsk to "Du får etterbetalt pensjon frå ".expr() + kravVirkDatoFom + ". Etterbetalinga blir vanlegvis betalt ut i løpet av sju yrkedagar. Vi kan trekke frå skatt og ytingar du har fått frå for eksempel Nav eller tenestepensjonsordningar. Derfor kan etterbetalinga di bli forsinka. Tenestepensjonsordninga har ni veker frist på å krevje trekk i etterbetalinga. Du kan sjekke eventuelle frådrag i utbetalingsmeldinga på $DITT_NAV.",
+                        English to "You will receive retroactive pension payments from ".expr() + kravVirkDatoFom + ". The retroactive payments will normally be made in the course of seven working days. We can make deductions for tax and benefits you have received, for example, from Nav or occupational pension schemes. Therefore, your retroactive payment may be delayed. Occupational pension schemes have a deadline of nine weeks to demand a deduction from the retroactive payments. You can check if there are any deductions from the payment notice at $DITT_NAV.",
+                    )
+                }
+                paragraph {
+                    text(
+                        Bokmal to "Hvis etterbetalingen gjelder tidligere år, trekker vi skatt etter skatteetatens standardsatser.",
+                        Nynorsk to "Dersom etterbetalinga gjeld tidlegare år, vil vi trekkje skatt etter standardsatsane til skatteetaten.",
+                        English to "If the retroactive payment refers to earlier years, we will deduct tax at the Tax Administration's standard rates.",
+                    )
+                }
+            }
+        }
+
+        // Arbeidsinntekt og pensjon
+        showIf(
+            regelverkType.isNotAnyOf(AlderspensjonRegelverkType.AP1967) and kravArsakType.isNotAnyOf(
+                KravArsakType.INSTOPPHOLD
+            )
+        ) {
+            // arbInntektAPOverskrift
+            title1 {
+                text(
+                    Bokmal to "Arbeidsinntekt og alderspensjon",
```

Comment:
Innholdet her og ut resten av brevet ser helt likt ut som i brevet VedtakEndringAvUttaksgrad. Ville laget felles fraser for hele denne blokken.